.github/workflows/ruby.yml

```diff
@@ -24,16 +24,8 @@ jobs:
         env:
           BUNDLE_GEMS__CONTRIBSYS__COM: ${{ secrets.BUNDLE_GEMS__CONTRIBSYS__COM }}
 
-      - name: Set up node
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16.20.0
-
-      - name: Install yarn dependencies and precompile assets
-        run: bundle exec rails assets:precompile
-
       - name: Rails tests
-        run: bundle exec rails test
+        run: CI=true bundle exec rails test
```

Comment:
I don't think this is necessary; it's set by default for Actions. Repos that do Codecov rely on it without setting it explicitly, e.g. https://github.com/CruGlobal/summer_missions/blob/d8e411dbbadd891985be2f8a5d94b2b1cccc45b7/spec/spec_helper.rb#L10

Reply:
Yep: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Sweet, thanks for verifying that!